source/jormungandr/jormungandr/interfaces/parsers.py

```diff
@@ -38,7 +39,10 @@
 
 float_gt_0 = parser_args_type.PositiveFloat()
 
-default_count_arg_type = parser_args_type.IntervalValue(min_value=0, max_value=1000)
+parser_min_count = app.config['PARSER_MIN_COUNT']
```

Comment:
I don't think it's worth having a `min_parameter`

(and it's not very important there, but It's better to create only one functionally-complete PR instead of 1 per file)

Reply:
It makes sense... I confess I hesitated.